bookx/test_settings.py

```diff
@@ -0,0 +1,10 @@
+from .settings import *             # NOQA
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
This may be faster than in Postgres, but consider still using Postgres for tests, or ALSO run tests in Postgres.
SQLite has subtly different behaviours features and some of them may mean getting back different results.
You can also run tests with `--reusedb` (https://pytest-django.readthedocs.io/en/latest/database.html#reuse-db-reuse-the-testing-database-between-test-runs) which should speed up test runs even in Postgres.